Products/Zuul/interfaces/eventclasses.py

```diff
@@ -75,6 +75,16 @@ def getTransformTree(uid):
         returns a transform for the event class context with all parent transforms
         """
 
+    def getTransformState(uid):
+        """
+        Returns a transform state.
+        """
+
+    def changeTransformState(uid):
+        """
+        Changes a transform state and returns result.
+        """
+
```

Comment:
I'd rather have an idempotent API here.  Calling a function that behaves like a toggle doesn't provide the API user much control.  Is  _transformState_ property the enabled flag? Or is there thought for other states?

If this is just for the enabled flag, then you it should be `isEnabled` as the getter and `setEnabled` for the setter (since we're generally following Java conventions in the Zenoss code base).  And `setEnabled` should take two arguments, the event UID and the boolean I'm setting the flag to.